Sources/PackageManagerDocs/Documentation.docc/UsingShellCompletion.md

```diff
@@ -0,0 +1,40 @@
+# Using shell completion scripts
+
+Customize your shell to automatically complete swift package commands.
+
+## Overview
+
+Package manager ships with completion scripts for Bash, ZSH, and Fish.
```

Comment:
Let's stay consistent with the capitalization chosen by Zsh developers per their website https://www.zsh.org

```suggestion
Package manager ships with completion scripts for Bash, Zsh, and Fish.
```

Reply:
thank you!